build_tools/customizations.rb

```diff
@@ -67,9 +67,7 @@ def dynamodb_example_deep_transform(subsegment, keys)
     api('CloudFront') do |api|
 
       api['shapes'].each do |_, shape|
-        if shape['members'] && shape['members']['MaxItems']
-          shape['members']['MaxItems']['shape'] = 'integer'
-        end
+        shape['members']['MaxItems']['shape'] = 'integer' if shape['members'] && shape['members']['MaxItems']
```

Comment:
>80 characters?

Reply:
Heck, I thought this was fixed - dang IDE. I'm on it!